src/math/random.js

```diff
@@ -62,62 +68,72 @@ p5.prototype.randomSeed = function(seed) {
 };
 
 /**
- * Return a random floating-point number.
- *
- * Takes either 0, 1 or 2 arguments.
+ * Returns a random number or a random element from an array.
  *
- * If no argument is given, returns a random number from 0
- * up to (but not including) 1.
+ * `random()` follows uniform distribution, which is like rolling a die. When
```

Comment:
Might be good to make it clear that uniform distribution _means_ that all numbers/array elements are equally likely, could be accomplished by combining these sentences or moving the dice example to the end of the paragraph.

Reply:
Agreed on mixing up the sequence. Also dropped the die example. Let me know if the latest version is too repetitive.